compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/platform/DesktopPlatformInput.desktop.kt

```diff
@@ -89,13 +89,8 @@ internal class DesktopTextInputService(private val component: PlatformComponent)
         currentInput?.value = newValue
     }
 
-    // TODO(https://github.com/JetBrains/compose-jb/issues/2040): probably the position of input method
-    //  popup isn't correct now
-    @Deprecated("This method should not be called, use BringIntoViewRequester instead.")
     override fun notifyFocusedRect(rect: Rect) {
```

Comment:
In the past, it was deprecated (as far as I remember during BTF2 introduction), and I created a CL to undeprecate it:
https://android-review.googlesource.com/c/platform/frameworks/support/+/2319924 for BTF1. For BTF2, they said it is a  wrong abstraction (also, the PlatformTextInputService was deprecated later, which is still [deprecated in AOSP](https://github.com/androidx/androidx/blob/121bef938b089f292caa9b6101f1e6397936f932/compose/ui/ui-text/src/commonMain/kotlin/androidx/compose/ui/text/input/TextInputService.kt#L141))

It is fine for me and it works. But be ready to discuss other methods to fix it in AOSP.

P.S.
We undeprecated `PlatformTextInputService` in the fork and we should upstream it as well. Or don't use it and deprecate again.